Dockerfile

```diff
@@ -1,46 +1,43 @@
-FROM baseruntime/baseruntime:latest
+FROM {{ config.docker.from }}
 
 # Description
 # Exposed ports:
 # * 53 - domain name system
 
-MAINTAINER Rado Pitonak <rpitonak@redhat.com>
+MAINTAINER {{ spec.maintainer }}
 
-ENV NAME=bind \
-    VERSION=0 \
-    RELEASE=1 \
-    ARCH=x86_64
+ENV NAME={{ spec.envvars.name }} \
+    VERSION={{ spec.envvars.version }} \
+    RELEASE={{ spec.envvars.release }} \
+    ARCH={{ spec.envvars.arch }}
```

Comment:
You should be able to use `config.os.arch` and so you wouldn't need to specify this in the spec file.